### DISCUSSION TITLE
how is default pixel size (transform?) determined when using rio.reproject?

### DISCUSSION THREAD

**Discussion by joemcglinchy** [corteva/rioxarray#640] [Q&A] [answered]
Hello,

I was wondering how is default pixel size (transform?) determined when using rio.reproject if none is specified? I see it is an optional argument.

**Comment by snowman2** [comment, marked as answer]
It is calculated using [calculate_default_transform](https://rasterio.readthedocs.io/en/stable/api/rasterio.warp.html#rasterio.warp.calculate_default_transform).
